terra-jupyter-r/Dockerfile

```diff
@@ -161,9 +150,17 @@ RUN mkdir -p /home/jupyter-user/.rpackages \
     "uuid"))' \
     && chown -R $USER:users /home/jupyter-user/.local
 
+## pip runs as jupyter-user
+ENV PIP_USER=true
+
+## After image installations, switch back to jupyter-user and create packages lib
+USER $USER
+RUN mkdir -p /home/jupyter-user/notebooks/packages \
+    && echo "R_LIBS=/home/jupyter-user/notebooks/packages" > /home/jupyter-user/.Renviron
 
 USER root
 
-RUN R -e 'IRkernel::installspec(user=FALSE)'
+RUN R -e 'IRkernel::installspec(user=FALSE)' \
+    && chown -R $USER:users /usr/local/lib/R/site-library
```

Comment:
@gpcarr Do the pre-installed packages (devtools, remotes, etc..) go into `/usr/local/lib/R/site-library`?

Reply:
yep 
<img width="1108" alt="Screen Shot 2020-06-24 at 2 43 09 PM" src="https://user-images.githubusercontent.com/23626109/85614848-5d182800-b629-11ea-965c-d065451ab83c.png">